.conf/dps_182/unbound.conf

```diff
@@ -1,81 +1,88 @@
+# https://nlnetlabs.nl/documentation/unbound/unbound.conf/
 server:
+	# Do not daemonize, to allow proper systemd service control and status estimation.
+	do-daemonize: no
+
+	# A single thread is pretty sufficient for home or small office instances.
+	num-threads: 1
+
+	# Logging: For the sake of privacy and performance, keep logging at a minimum!
+	# - Verbosity 2 and up practically contains query and reply logs.
 	verbosity: 0
+	log-queries: no
+	log-replies: no
+	# - If required, uncomment to log to a file, else logs are available via "journalctl -u unbound".
+	#logfile: "/var/log/unbound.log"
 
+	# Set interface to "0.0.0.0" to make Unbound listen on all network interfaces.
+	# Set it to "127.0.0.1" to listen on requests from the same machine only, useful in combination with Pi-hole.
 	interface: 0.0.0.0
+	# Default DNS port is "53", mDNS is "5353", so "5335" is best in combination with Pi-hole.
 	port: 53
```

Comment:
is the port set correctly? Should it be `53` or `5335`

Reply:
`53` by default, `5335` when Pi-hole is installed.

Reply:
got it